src/classes/CON_ContactMerge.cls

```diff
@@ -126,15 +126,22 @@ public class CON_ContactMerge {
         
         //get loser contact Accounts that should be merged out.  This includes
         // One2One accounts and Solo Household Accounts
-        loserAccountsToBeMerged = [Select Id,npe01__SYSTEMIsIndividual__c,npe01__SYSTEM_AccountType__c,Name from Account a
-            where Id IN :loserAccountIds and 
-            (npe01__SYSTEM_AccountType__c=:CAO_Constants.ONE_TO_ONE_ORGANIZATION_TYPE or
-                (npe01__SYSTEM_AccountType__c=:CAO_Constants.HH_ACCOUNT_TYPE and Number_of_Household_Members__c = 1))
-            and Name<>:CAO_Constants.BUCKET_ACCOUNT_NAME];        
```

Comment:
The check for `and Name<>:CAO_Constants.BUCKET_ACCOUNT_NAME` has been removed here. Can there be an Account with `ONE_TO_ONE_ORGANIZATION_TYPE` or `HH_ACCOUNT_TYPE` but with `CAO_Constants.BUCKET_ACCOUNT_NAME` Name?
@njjc , @force2b

Reply:
The CAO constants attach to bucket account will only attach to an account that meets the system account type criteria so even if we have an Account with the same name it's not going to be used as a bucket account in our code.